### ISSUE TITLE
Menu module duplication

### ISSUE THREAD

**Issue by rinka88** [templaza/astroid-framework#701]
When creating a menu using Astroid, the module menu and displaying it through the module position is duplicated at the top.
Astroid 3.0.15
Template two 1.0.4
Joomla 5.1.1
![IMG_20240617_094804_370](https://github.com/templaza/astroid-framework/assets/40716604/0f2c0b7d-aadf-445a-a06f-e090aea64ad0)

**Comment by sonvnn** [comment]
Can you send us a website link or details of what you did to create this problem?
It is very difficult to give any advice on abstract/unclear images and problems like this.

**Comment by rinka88** [comment]
Sent the link and access to your email

**Comment by rinka88** [comment]
![IMG-20240617-WA0013](https://github.com/templaza/astroid-framework/assets/40716604/8df0a57d-9317-4d7f-ac63-d1c7ad18d2ef)
Extra menu code is added to the top of the page. (shown by red lines)

**Comment by sonvnn** [comment]
@rinka88 Please send me your website via sonlv@templaza.com
Thanks,
Sonny

**Comment by rinka88** [comment]
> @rinka88 Please send me your website via [sonlv@templaza.com](mailto:sonlv@templaza.com) Thanks, Sonny

I already sent it to you yesterday)

**Comment by Joomlaplates** [comment]
This happens when the header is renamed then the Astroid create a new header section.

**Comment by rinka88** [comment]
> Это происходит, когда заголовок переименовывается, а затем Astroid создает новый раздел заголовка.

how to fix this?

**Comment by Joomlaplates** [comment]
Go to the layout option and delete the double header.

**Comment by rinka88** [comment]
> @rinka88 Please send me your website via [sonlv@templaza.com](mailto:sonlv@templaza.com) Thanks, Sonny

did you manage to watch?

**Comment by sonvnn** [comment]
@rinka88 I still did not received your email yet. Check spam mail also but nothing.

**Comment by sonvnn** [comment]
I got it. I will check when I have free time, tomorrow. Thanks

**Comment by sonvnn** [comment]
@rinka88 System raise error "Username and password do not match or you do not have an account yet."

**Comment by rinka88** [comment]
> @rinka88 System raise error "Username and password do not match or you do not have an account yet."

Try again. looks like it should work

**Comment by sonvnn** [comment]
@rinka88 Because you set wrong mode for Mobile Menu Module mode. You set it is Site instead of Mobile.
![Screen Shot 2024-06-20 at 10 23 36](https://github.com/templaza/astroid-framework/assets/3204103/e7ec1f1e-1ac3-47c4-b518-2192929c52d9)

Correct is 
![Screen Shot 2024-06-20 at 10 23 36](https://github.com/templaza/astroid-framework/assets/3204103/be89a798-8bfd-4226-a812-b1ec69b66fd1)

I set it correct in КАЗАК (Казах) language. You can set another in Russian and check your site again.

Best Regards,
Sonny

**Comment by rinka88** [comment]
Thank you very much. I forgot about this setting. Now it worked.

**Comment by rinka88** [comment]
> @rinka88 Because you set wrong mode for Mobile Menu Module mode. You set it is Site instead of Mobile.
> 
> ![Screen Shot 2024-06-20 at 10 23 18](https://private-user-images.githubusercontent.com/3204103/341258287-2709d139-f01a-49ab-b867-0ff65e66779d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5ODI2NjksIm5iZiI6MTcxODk4MjM2OSwicGF0aCI6Ii8zMjA0MTAzLzM0MTI1ODI4Ny0yNzA5ZDEzOS1mMDFhLTQ5YWItYjg2Ny0wZmY2NWU2Njc3OWQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjFUMTUwNjA5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjA4MDhiNTQyMDY0YmE5ZGU5ZTYwZTM4NmMxZTc3Y2UxMmQ3NDMwMjE3Mjc5ZWExMTYyMDFiMmQ2MzZhYWU5MCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.D7QleSy34Yb5XoelO8oBhoBlF-7Ucui-FAZBNmNq_2Q)
> 
> Correct is ![Screen Shot 2024-06-20 at 10 23 36](https://private-user-images.githubusercontent.com/3204103/341257961-be89a798-8bfd-4226-a812-b1ec69b66fd1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5ODI2NjksIm5iZiI6MTcxODk4MjM2OSwicGF0aCI6Ii8zMjA0MTAzLzM0MTI1Nzk2MS1iZTg5YTc5OC04YmZkLTQyMjYtYTgxMi1iMWVjNjliNjZmZDEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjFUMTUwNjA5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGM5ZDkyYTIzNmQxYWZiMGExYWRkMmI1OGQyMmYwMzljZDhkYjkxNGY4ZTYyMjc0MDVkMzJmZmJkMWE4ZTBiOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.GnRWWflr2kvQ9WMvEpVxLrALWqxhK-q5BHEdnXySoKU)
> 
> I set it correct in КАЗАК (Казах) language. You can set another in Russian and check your site again.
> 
> Best Regards, Sonny

I think this should be added to the documentation. So that others can find out and not repeat it.

**Comment by sonvnn** [comment]
Yes @Jessglynne will update it on documentation. We are trying update documentation, but Human resource is limited.

**Comment by felixkat** [comment]
Closing as issue marked as resolved.
